prodactivity/managers/RoutineManager.tsx

```diff
@@ -12,14 +13,13 @@ export default class RoutineManager {
      */
     async getRoutines(): Promise<RoutineViewModel[] | undefined> {
         try {
-            let response = await fetch(`${this.baseURL}`);
-            let jsonRoutines = await response.json();
-            
-            var routines: RoutineViewModel[] = [];
+            const jsonRoutines = await FetchHelper.get(this.baseURL);
+
+            const routines: RoutineViewModel[] = [];
```

Comment:
If you're declaring it as an array type you do not need to explicitly set it equal to an empty array, unless you want to free country *shrug*